ASTree.cpp

```diff
@@ -3287,13 +3287,16 @@ void print_src(PycRef<ASTNode> node, PycModule* mod)
         break;
     case ASTNode::NODE_TERNARY:
         {
-            /* parenthesis might not be needed,
-             * but when if-expr is part of numerical expression, ternary has the LOWEST precedence
+            /* parenthesis might be needed
+             * 
+             * when if-expr is part of numerical expression, ternary has the LOWEST precedence
              *     print(a + b if False else c)
              * output is c, not a+c (a+b is calculated first)
+             * 
+             * but, let's not add parenthesis - to keep the source as close to original as possible in most cases
```

Comment:
To clarify, I'm fine with adding parentheses here (ideally we could try to detect if they're necessary, but that's more work).  My suggestion was just to remove the extra whitespace inside the parens, since it's inconsistent with parenthesis output everywhere else.